velox/exec/fuzzer/MemoryArbitrationFuzzer.cpp

```diff
@@ -855,6 +850,22 @@ void MemoryArbitrationFuzzer::verify() {
     }
   });
 
+  // Create a thread that randomly abort one worker thread
+  // every seconds_to_abort seconds.
+  std::thread abortControlThread([&]() {
+    if (FLAGS_task_abort_interval_ms == 0) {
+      return;
+    }
+    while (!stop) {
+      std::this_thread::sleep_for(
+          std::chrono::milliseconds(FLAGS_task_abort_interval_ms));
+      auto tasksList = Task::getRunningTasks();
+      auto index = getRandomIndex(rng_, tasksList.size() - 1);
+      ++taskAbortRequestCount;
+      tasksList[index]->requestAbort();
+    }
```

Comment:
Shall we have error handling in this thread, by having a try and catch all block

Reply:
Also we can consider to use folly:FunctionScheduler to achieve the periodic scheduling of this task. It provides better functionality and encapsulations.